client/src/actions/postingsActions.js

```diff
@@ -4,15 +4,15 @@ export default function fetchPostings(searchKeywords) {
     dispatch({
       type: 'FETCH_POSTINGS_REQUEST'
     });
-    console.log("Fetching search results");
+    console.log('Fetching search results');
     return fetch('http://localhost:5000/search', {
-        method: 'POST',
-        headers:{
-          Accept: 'application/json',
-          'Content-Type': 'application/json',
-        },
-        body: `{"keywords": "${searchKeywords}"}`,
-      })
+      method: 'POST',
+      headers:{
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: `{'keywords': '${searchKeywords}'}`,
```

Comment:
This is a JSON literal, so I don't think it would be allowed to have single quotes.